sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -950,12 +943,10 @@ def store_callback(arg):
         with open(file_name_2, 'w') as f:
           json.dump(json_output, f)
 
-      res = mock.Mock()
-      res.insertErrors = []
```

Comment:
Why the change to return value ?

Reply:
because `insert_rows_json` returns a list of errors (empty when no errors), and `client.tabledata.InsertAll` returns an object that contains the list of errors within it